test/Microsoft.AspNet.Mvc.FunctionalTests/TagHelpersTest.cs

```diff
@@ -30,6 +30,7 @@ public class TagHelpersTests
         [InlineData("Index")]
         [InlineData("About")]
         [InlineData("Help")]
+        [InlineData("Environment")]
```

Comment:
don't extend this class to test MVC tag helpers.  it's about a group of custom tag helpers, not our out-of-the-box ones.  so extend `MvcTagHelpersTests` instead.